Testcases/AlarmTests/ahuFailedToTurnOffAlarmTest.robot

```diff
@@ -0,0 +1,73 @@
+*** Settings ***
+Documentation          This testcase validates Alarm-01 AHU Failed to Turn OFF Alarm Automated Test Steps
+...                     Environment: ${host_ip}
+...                     Group Name: ${group_name}
+#...                    Created on 20th Dec 2021
+Resource    ${EXECDIR}/Resources/apiresources.robot
+Resource    ${EXECDIR}/Resources/uiresources.robot
+Resource    ${EXECDIR}/Resources/connection.robot
+Resource    ${EXECDIR}/Resources/AlarmTestsResources/ahuFailedToTurnOffAlarmResources.robot
```

Comment:
alarmResources can be a single generic file which contains keyword that are used by all 10 alarm testcases.
Most of the steps are repeating in all 10 alarm testcases.

Reply:
done